app/root.tsx

```diff
@@ -16,11 +15,11 @@ import { CartOpenContextProvider } from '~/components/cart/cart-open-context';
 import '~/styles/index.scss';
 
 export async function loader() {
-    return json({
+    return {
```

Comment:
why `json` was here?

Reply:
With `json` from '@remix-run/node' app won't run in codux because of a problem I wrote in a description.